### PR TITLE
replace CircleCI with GitHub Actions

### PR DESCRIPTION
**Pull request by mavendonovanhubbard**

### CODE REVIEW

**Review by diclophis — APPROVED** [review]
yea looks good

File: spec/lib/mavenlink/workspace_spec.rb [hunk 1]
```diff
@@ -410,21 +410,17 @@
...
         specify do
           expect do
-            begin
-                     subject.save!
-            rescue StandardError
-              nil
-                   end
+            subject.save!
+          rescue StandardError
+            nil
```
Comment (arthurlm44) [inline spec/lib/mavenlink/workspace_spec.rb:415]: I would think the `begin` block is necessary here. Otherwise doesn't it always return `nil`?

Comment (mavendonovanhubbard) [reply spec/lib/mavenlink/workspace_spec.rb:415]: Rubocop actually made me change it to that. https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Style/RedundantBegin

Comment (diclophis) [reply spec/lib/mavenlink/workspace_spec.rb:415]: yea, there might be a special syntax case for rescue without begin...

you can also do stuff like

```
def thang(args)
  # ... stuff that might raise exception
rescue ExceptionBit
  # ... yadda
end
```

**Pull request merged**